dot/state/epoch_test.go

```diff
@@ -401,12 +401,24 @@ func TestStoreAndFinalizeBabeNextEpochData(t *testing.T) {
 			err := epochState.blockState.db.Put(headerKey(tt.finalizeHash), []byte{})
 			require.NoError(t, err)
 
+			// before finalize next epoch data this epoch should not be defined
+			definedEpochData, definedConfigData, err := epochState.AlreadyDefined(tt.finalizeEpoch)
+			require.NoError(t, err)
+			require.False(t, definedEpochData)
+			require.False(t, definedConfigData)
+
 			err = epochState.FinalizeBABENextEpochData(tt.finalizeEpoch)
 			if tt.expectErr != nil {
 				require.ErrorIs(t, err, tt.expectErr)
 			} else {
 				require.NoError(t, err)
 
+				// after finalize next epoch data this epoch should not defined
```

Comment:
nit I think `for` is missing here?

```suggestion
				// after finalize next epoch data for this epoch should not defined
```